tsconfig.json

```diff
@@ -20,10 +20,9 @@
     "isolatedModules": true,
     "useDefineForClassFields": false,
     "strict": true,
-    "sourceMap": true,
```

Comment:
Any specific reason to remove sourceMap?

Reply:
It was a duplicate.